<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>300</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="3">
       <widget class="QLabel" name="physBiorythmName">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Физиологический биоритм</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="psychoBiorythmName">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Психологический биоритм</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="intellBiorythmName">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Интеллектуальный биоритм</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="physBiorythm">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="6">
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>27</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="psychoBiorythm">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5" rowspan="4">
       <widget class="QPushButton" name="buildGraphic">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Построить график</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="intellBiorythm">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
          <property name="shortcut">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
          <property name="shortcut">
           <string>Right</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="currDayLabelName">
        <property name="maximumSize">
         <size>
          <width>175</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Дата расчёта:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="currDayLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="birthdayLabelName">
        <property name="maximumSize">
         <size>
          <width>175</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Дата рождения:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="birthdayLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTextEdit" name="textEdit">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="separator"/>
    <addaction name="setCurrDate"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="cursorTrackingSwitch"/>
    <addaction name="action"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Формат ввода-вывода</string>
    </property>
    <addaction name="separator"/>
    <addaction name="standartFormat"/>
    <addaction name="textFormat"/>
    <addaction name="reverseFormat"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Показать биоритмы</string>
    </property>
    <addaction name="physShowSwitch"/>
    <addaction name="psychoShowSwitcch"/>
    <addaction name="intellShowSwitch"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Критические точки</string>
    </property>
    <addaction name="maxLineSwitch"/>
    <addaction name="maxCrossSwitch"/>
    <addaction name="minLineSwitch"/>
    <addaction name="minCrossSwitch"/>
    <addaction name="nullSwitch"/>
    <addaction name="critMinSwitch"/>
    <addaction name="critMaxSwitch"/>
    <addaction name="critNullSwitch"/>
    <addaction name="action_8"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Полноэкранный режим</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="standartFormat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Стандартный</string>
   </property>
  </action>
  <action name="textFormat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Текстовый</string>
   </property>
  </action>
  <action name="reverseFormat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Обратный &quot;американский&quot;</string>
   </property>
  </action>
  <action name="physShowSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Физиологический биоритм</string>
   </property>
  </action>
  <action name="psychoShowSwitcch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Психологический биоритм</string>
   </property>
  </action>
  <action name="intellShowSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Интеллектуальный биоритм</string>
   </property>
  </action>
  <action name="setCurrDate">
   <property name="text">
    <string>Рассмотреть сегодняшнюю дату</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сглаживание</string>
   </property>
  </action>
  <action name="action_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Толстые линии биоритмов</string>
   </property>
  </action>
  <action name="maxLineSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Пометить точки максимума прямыми</string>
   </property>
  </action>
  <action name="maxCrossSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Пометить точки максимума крестами</string>
   </property>
  </action>
  <action name="minLineSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Пометить точки минимума прямыми</string>
   </property>
  </action>
  <action name="minCrossSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Пометить точки минимума крестами</string>
   </property>
  </action>
  <action name="nullSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Пометить точки неустойчивости окружностями</string>
   </property>
  </action>
  <action name="critMinSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Пометить критические минимумы</string>
   </property>
  </action>
  <action name="critMaxSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Пометить критические максимумы</string>
   </property>
  </action>
  <action name="critNullSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Пометить точки критической неустойчивости</string>
   </property>
  </action>
  <action name="action_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Пометить дату рождения</string>
   </property>
  </action>
  <action name="cursorTrackingSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Отслеживание курсора</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
